### PR TITLE
Update brute force docs

### LINKED ISSUE #27378: update brute force docs to reflect available lockouts modes (temporary / permanent / mixed)
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

docs

### Describe the bug

We should update brute force docs to reflect changes made by https://github.com/keycloak/keycloak/pull/26208
Docs show that image of brute force admin UI:

image

But the UI has changed, as below:

image

image

image

image

### Version

24.0.0

### Regression

- [ ] The issue is a regression

### Expected behavior

N/A

### Actual behavior

N/A

### How to Reproduce?

Build main branch and see brute force admin UI

### Anything else?

I could see that because I am working on #25917

### PR DESCRIPTION
**Pull request by gilvansfilho**
Fixes #27378

### CODE REVIEW

**Comment by gilvansfilho** [comment]
@andymunro @rmartinc @douglaspalmer 
Please, could you review?

**Review by douglaspalmer — APPROVED** [review]
This is great, thank you. I'm approving as is but would it be possible to add images for each Brute Force Mode?

**Comment by gilvansfilho** [comment]
@douglaspalmer done...

File: docs/documentation/server_admin/topics/threat/brute-force.adoc [hunk 1]
```diff
@@ -104,11 +151,11 @@ By multiples strategy, wait time is incremented when the number (or count) of fa
 |**10** |**30** | 5 | **60**
 |===
 
-At the fifth failed attempt of the `Effective Wait Time`, the account is disabled for `30` seconds. After reaching the next multiple of `Max Login Failures`, in this case `10`, the time increases from `30` to `60` seconds. 
+At the fifth failed attempt the account is disabled for `30` seconds. After reaching the next multiple of `Max Login Failures`, in this case `10`, the time increases from `30` to `60` seconds.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/threat/brute-force.adoc:154]:
```suggestion
At the fifth failed attempt, the account is disabled for `30` seconds. After reaching the next multiple of `Max Login Failures`, in this case `10`, the time increases from `30` to `60` seconds. 
```

File: docs/documentation/server_admin/topics/threat/brute-force.adoc [hunk 2]
```diff
@@ -125,33 +172,88 @@ For linear strategy, wait time is incremented when the number (or count) of fail
 |**10** |**30** | 5 | **180**
 |===
 
-At the fifth failed attempt for the `Effective Wait Time`, the account is disabled for `30` seconds. Each new failed attempt increases wait time.
+At the fifth failed attempt the account is disabled for `30` seconds. Each new failure increases wait time according value specified at `wait increment`.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/threat/brute-force.adoc:175]:
```suggestion
At the fifth failed attempt, the account is disabled for `30` seconds. Each new failure increases wait time according value specified at `wait increment`.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good. Just a few changes to suggest, @gilvansfilho

**Review by rmartinc — APPROVED** [review]
The changes LGTM. Thanks @gilvansfilho! 
Perform the modifications requested by @andymunro and we are good with it.

**Review by ahus1 — APPROVED** [review]
Approving as of @rmartinc's review

**Comment by gilvansfilho** [comment]
@andymunro @rmartinc PR was merged before I made the requested changes. Should I submit a new PR?

**Comment by ahus1** [comment]
@gilvansfilho - I've accepted @andymunro's review comments (which were about adding commas) via the GitHub UI which I could do as a maintainer. You now see 3 commits associated with this PR. Those were then squashed when I merged them.

So the work should be completed, and no additional tasks are necessary.

**Pull request merged**